projects/backend/capi/articles.ts

```diff
@@ -31,7 +32,7 @@ type NotInCAPI =
     | 'mediaType'
     | 'slideshowImages'
 
-type OptionalInCAPI = 'kicker' | 'bylineImages' | 'trail'
+type OptionalInCAPI = 'kicker' | 'bylineImages' | 'trail' | 'articleType'
```

Comment:
I can see this type was there already but I'm not sure what `OptionalInCapi` means in this context? It sounds like it's an optional filed in CAPI, but those don't look like CAPI fields that I recgonize?